lib/api_catalogue.rb

```diff
@@ -10,6 +12,28 @@ def self.from_csv(catalogue_csv:, organisation_csv:)
     new(apis: apis, organisations: organisations)
   end
 
+  def self.from_url(url)
+    V1AlphaFetcher.fetch(url)
+  rescue RuntimeError => e
+    puts e.message
+    ApiCatalogue.new(apis: [], organisations: [])
+  end
+
+  def self.merge(*api_catalogues)
+    all_apis = []
+    all_orgs = []
+    api_catalogues.each do |catalogue|
```

Comment:
could also use `map` here

Reply:
as discussed offline - going to leave it as `each` for readability